### PR TITLE
upgpkg: bun add aarch64 build

### PR DESCRIPTION
**Pull request by Toby222**

### CODE REVIEW

File: bun/PKGBUILD [hunk 1]
```diff
@@ -3,15 +3,21 @@ pkgname=bun
 pkgver=0.1.5
 pkgrel=1
```
Comment (navarroaxel) [inline bun/PKGBUILD]:
```suggestion
pkgrel=2
```

**Review by navarroaxel — APPROVED** [review]

File: bun/PKGBUILD [hunk 2]
```diff
@@ -1,17 +1,23 @@
...
-arch=(x86_64)
+arch=('x86_64' 'aarch64')
 url="https://github.com/Jarred-Sumner/bun-releases-for-updater"
 license=(MIT)
 provides=(bun)
-source=(${pkgname}-${pkgver}.zip::$url/releases/download/$pkgname-v$pkgver/$pkgname-linux-x64.zip)
-sha256sums=('524a2d7e51ddda4786799552ae1c18ab8e6173bb30f158f26cae899a7e74f22f')
+source_x86_64=(${pkgname}-${pkgver}.zip::$url/releases/download/$pkgname-v$pkgver/$pkgname-linux-x64.zip)
+sha256sums_x86_64=('524a2d7e51ddda4786799552ae1c18ab8e6173bb30f158f26cae899a7e74f22f')
+source_aarch64=(${pkgname}-${pkgver}.zip::$url/releases/download/$pkgname-v$pkgver/$pkgname-linux-aarch64.zip)
```
Comment (navarroaxel) [inline bun/PKGBUILD]: All filenames should be different to allow the usage of `makepkg --allsource` for testing.
```suggestion
source_aarch64=(${pkgname}-${pkgver}-aarch64.zip::$url/releases/download/$pkgname-v$pkgver/$pkgname-linux-aarch64.zip)
```

Comment (Toby222) [reply bun/PKGBUILD]: Good to know. First time in a PKGBUILD ^^;

**Comment by navarroaxel** [comment]
Package released!

**Comment by Toby222** [comment]
Now that I look at it again... Why is `bun` copied to `/usr/lib/bun/bun`?

**Comment by navarroaxel** [comment]
> Now that I look at it again.. Why is it `bun` copied to `/usr/lib/bun/bun`?

Yes!, it's copied and symlinked to the `bin/` folder.
image

image

**Comment by Toby222** [comment]
Why that instead of installing the file direactly as `/usr/bin/bun`?

**Comment by navarroaxel** [comment]
> Why that instead of installing the file direactly as `/usr/bin/bun`? confounded

It's a harmless difference, you can just put the binary in `/usr/bin` or use a symlink this way. The `nodejs` package put the binary in `/usr/bin`, the `signal-desktop` symlinks to `/usr/lib/signal-desktop/signal-desktop`. Just talking about official packages.

My `webtorrent-cli` AUR package uses symlink because in `/usr/lib` the app has the `node_modules` dependencies.

**Pull request merged**